app/dashboard/templates/dashboard/sponsors.html

```diff
@@ -172,38 +176,33 @@
                       </div>
                       <div role="tablist">
                         <b-card no-body class="my-3" v-for="(prize, prizeIndex) in prizes" v-if="prizes">
-                          <b-card-header header-tag="header" class="row mx-0 px-4 py-2 bg-white" role="tab">
+                          <b-card-header header-tag="header" class="justify-content-between row mx-0 px-4 py-2 bg-white" role="tab">
 
                             <div class="col-12 col-md-5 col-lg-6 text-center text-md-left pl-md-0">
                               <div class="my-2">
                                 <p class="m-0">
                                   <a target="_blank" :href="prize.url" class="font-body font-weight-semibold d-inline-block text-black">
                                     [[`${prize.title.slice(0, 100)}${prize.title.length > 100 ? '...' : ''}`]]
                                   </a>
+                                  <span v-if="prize.value_eth" class="tag token inner-tooltip">[[prize.value_eth | decimals]] ETH</span>
+                                  <span v-if="prize.values_usdt" class="tag usd inner-tooltip">[[prize.values_usdt]] USD</span>
```

Comment:
^ Zoek could you revert this to how this was ?
Cause we had issues where this token would break into multiple lines if the title was long

Reply:
done :)